test/data/vessel-valid/polars.json

```diff
@@ -0,0 +1,349 @@
+{
+  "uuid": "urn:mrn:signalk:uuid:c0d79334-4e25-4245-8892-54e8ccc8021d",
+  "polars": [
+    {
+      "91c8d96d-4bbd-4c2f-ab36-295b04e83589": {
```

Comment:
This extra layer does not make sense or I am missing something: either we have objects in the array where all the data, including the uuid is in the object's properties, or have a container object with uuids as property keys.

Reply:
Imho the array level should be removed so that `polars` is an object. See how it is in my comment https://github.com/SignalK/specification/pull/360#pullrequestreview-584842629

This would allow retrieving individual polars with paths like `.../polars/91c8d96d-4bbd-4c2f-ab36-295b04e83589`.

Or am I missing something?